### ISSUE TITLE
Popup menus displayed as single window

### ISSUE THREAD

**Issue by onny** [Cloudef/orbment#63]
![loliwm-popup-menu](https://cloud.githubusercontent.com/assets/757752/6410784/06f4b086-be6a-11e4-9f97-2764609c5b9b.png)
Screenshot of Nautilus, right clicking on a folder.

Btw. thank you for loliwm. Its the first tiled window manager for wayland which I'm using for some weeks now and I'm really happy with it. I hope we can get rid of the few bugs left :)

**Comment by Cloudef** [comment]
Interesting. Is this using wayland GTK backend or running as xwayland program?

**Comment by onny** [comment]
No, I was not able to use xprop on Nautilus so I guess its a native Wayland application. I also defined this in my .zlogin file:
```
GDK_BACKEND=wayland
CLUTTER_BACKEND=wayland
#QT_QPA_PLATFORM=wayland-egl
#QT_STYLE_OVERRIDE=gtk
#export QT_STYLE_OVERRIDE
#export QT_QPA_PLATFORM
export GDK_BACKEND
export CLUTTER_BACKEND
export XKB_DEFAULT_LAYOUT=de
export TERMINAL=urxvt
[[ -z $DISPLAY && $XDG_VTNR -eq 1 ]] && exec dbus-launch loliwm --prefix logo
```

**Comment by Cloudef** [comment]
All right, I install nautilus and see what's going on.

**Comment by Cloudef** [comment]
Nautilus seems to use xdg_surface instead of xdg_popup for the file right click menus for some reason. The left-panel right click menus work correctly. I report this when I'm done with my wlc refactor work.

**Comment by onny** [comment]
Good news! Seems to be fixed now in gnome 3.16 (even though its not stable yet here in ArchLinux)
![orbment-2015-03-29t01 07 32z](https://cloud.githubusercontent.com/assets/757752/6883674/8246d8ba-d5b0-11e4-99ed-e307273efe90.png)
Thank you for having a look into this issue.

**Comment by Cloudef** [comment]
Oh nice :) Closing then.
